yt/yt_proto/yt/client/api/rpc_proxy/proto/api_service.proto

```diff
@@ -3368,6 +3368,7 @@ message TRspGetQueryTrackerInfo
     required string cluster_name = 1;
     required bytes supported_features = 2; // YSON
     repeated string access_control_objects = 3;
+    required string query_tracker_stage = 4;
```

Comment:
Will it work with `required`? If only QT is updated or only proxy

Reply:
I suppose we may have ones in case of new proxies, but you can check for sure

Reply:
Tested. It doesn't break, just returns `"query_tracker_stage":""`

Funny:
https://protobuf.dev/programming-guides/proto2/#field-labels
<img width="735" alt="Screenshot 2024-09-04 at 13 54 22" src="https://github.com/user-attachments/assets/afdf6aa3-3e51-4b38-b632-577f777cc003">

I will change it to optional